enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/load_balancing/plugins/server_policies/Policies.java

```diff
@@ -31,7 +33,7 @@
 {
     public static final String POLICY_KEY = "load_balancing.policy"; // TODO: move somewhere (driver support package?)
```

Comment:
Can we change this key to something else?
One plan is to pass routing context via bolt url, in we go down that road, then it is really a bad idea to have a `.` in the parameters as it requires a skip in url.

I would vote for `policy` or `load_balancing_policy`.